pallets/api/src/fungibles/benchmarking.rs

```diff
@@ -0,0 +1,22 @@
+//! Benchmarking setup for pallet-parachain-template
```

Comment:
No need if we are just wrapping to start, suggest removing until it is actually required.